lib/avrora/codec/object_container_file.ex

```diff
@@ -19,8 +19,20 @@ defmodule Avrora.Codec.ObjectContainerFile do
   alias Avrora.Resolver
   alias Avrora.Schema.Encoder, as: SchemaEncoder
 
+  @impl true
+  def decodable?(payload) when is_binary(payload) do
+    case payload do
+      <<@magic_bytes, _::binary>> -> true
+      _ -> false
+    end
+  end
+
   @impl true
   def is_decodable(payload) when is_binary(payload) do
+    Logger.warning(
+      "`Avrora.Codec.ObjectContainerFile.is_decodable/1` is deprecated, use `Avrora.Codec.ObjectContainerFile.decodable?/1` instead"
+    )
+
     case payload do
```

Comment:
Could you please replace the code with a call of `decodable?` instead of duplication